config_alerts_notify.go

```diff
@@ -8,7 +8,9 @@ import (
 
 // NotifyConfig hold logging and email settings for the notify package.
 type NotifyConfig struct {
-	SMTP *SMTPConfig `bson:"smtp" json:"smtp" yaml:"smtp"`
+	NotificationsTarget       int         `bson:"notifications_target" json:"notifications_target" yaml:"notifications_target"`
```

Comment:
I think these could have better names, should definitley say "BufferTargetPerInterval" or some such.

Reply:
done